TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleop/BasicMotorTest.java

```diff
@@ -10,35 +10,44 @@
 /**
  * TeleOp OpMode for testing motors.
  */
+// "BasicMotorTest" is the name of OpMode in Driver Station
 @TeleOp(name = "BasicMotorTest", group = "Iterative OpMode")
+
 public class BasicMotorTest extends OpMode {
     private final ElapsedTime time = new ElapsedTime();
-    HWC robot;
+    HWC robot; // Declares roboBt as an instance of a hardware class.
```

Comment:
Comment On line above

Reply:
Plus, it should start with `private`, enough though it's implied.

Reply:
It should start with `private`, enough though it's implied.